app/storage/index.ts

```diff
@@ -115,10 +125,42 @@ export const useStorage = create<Storage>()(
           delete state.pendingUserOpLog[pendingUserOpLog.id]
         })
       },
+      updateBalance: async (accountId: string, balance: HexString) => {
+        await set(({ state }) => {
+          state.account[accountId].balance = balance
+        })
+      },
```

Comment:
I would suggest using `bigint` for the typing of `balance`, and convert it when setting to storage.

```suggestion
      updateBalance: async (accountId: string, balance: bigint) => {
        await set(({ state }) => {
          state.account[accountId].balance = number.toHex(balance)
        })
      },
```

Reply:
If using number.toHex(), should we switch to BigNumberish?